templates/scenarios/csharp/message-extension/Bot/TeamsMessageExtension.cs.tpl

```diff
@@ -194,9 +194,17 @@ public class TeamsMessageExtension : TeamsActivityHandler
             Images = new List<CardImage> { new CardImage("https://raw.githubusercontent.com/microsoft/botframework-sdk/master/icon.png") },
         };
 
+        // By default the link unfurling result is cached in Teams for 30 minutes.
+        // The code has set a cache policy and removed the cache for the app.
+        var action = new CardAction
```

Comment:
Can we link the doc from here as well, e.g.,

```
// By default the link unfurling result is cached in Teams for 30 minutes.
 // The code has set a cache policy and removed the cache for the app. Learn more here: https://learn.microsoft.com/microsoftteams/platform/messaging-extensions/how-to/link-unfurling?tabs=dotnet%2Cadvantages#remove-link-unfurling-cache
```

Reply:
what's the recommendation of Production? Does ME suggest enable cache for Production code?
If that's only true for debugging, is it better to put this part in README instead of template code? @JerryYangKai  @MuyangAmigo

Reply:
@eriolchan From official document there is no suggest environment for enable cache. It depends on the customers need.